doc/authentication.md

```diff
@@ -0,0 +1,47 @@
+
+Cockpit Authentication
+================================
+
+Cockpit authorizes users by looking at the Authorization header. The contents
```

Comment:
Sorry was confused. It is ```Authorization``` (I hate HTTP naming) ... but lets still mention the ```/login``` request.